src/general/leading.md

```diff
@@ -0,0 +1,46 @@
+# General Leading
+Suit 0135, NT 024th
+
+## Opening Leading
+Against Suit Contract
+- 0135
+- Trump leads: count (low even, high odd, but preserve the highest spot card except to lead highest of equals)
+
+Against NT Contract
+- 024th
+- Lead A or K
+- Honor leads: ace requests unblock or count signal; queen requests jack; highest equal from non-ace sequences and interior sequences
+
+
+## Later leading
+Against suit contracts:
+- Honor leads: king from ace-king; otherwise, highest equal from sequences and interior sequences
+- Spot-card leads in opening leader's suit: high from remaining doubleton; low from remaining tripleton
+- Spot-card leads in a new suit: third highest from even length; lowest from odd length
+- Trump leads: count (low even, high odd, but preserve the highest spot card except to lead highest of equals)
+Against notrump contracts:
+- Honor leads: highest equal from sequences and interior sequences
+- Spot-card leads in opening leader's suit: original fourth highest
+- Spot-card leads in a new suit: attitude
+
+
+## Lead A or K
+### High level Contract
+- Lead A ask for encourage
+- Lead K ask for count
+
+When defensing a high level (over 5) contract,
+we need to cash AK if possible and distinguish the suit count.
+So when we lead with A, partner must tell us he has K or not.
+If lead with K, partner must tell us the count so we can decide to cash out or switch.
+
+
+## Partner's suit
+We lead `3rd/5th` in partner's suit so he could read our card count.
```

Comment:
I have an agreement with lorderic before. Lets have a discussion over that later.

Reply:
ok, mark as `?`